### ISSUE TITLE
usemin option uglifyjs error "Maximum call stack size"

### ISSUE THREAD

**Issue by vijeetgv** [ericclemmons/grunt-angular-templates#123]
Hi,
when I try the usemin option it seems to be failing:
Gruntfile code:
<pre>
ngtemplates:  {
      app: {
        src: '<%= yeoman.app %>/templates/**/*.html',
        dest: '.tmp/templates.js', 
        options:  {
          htmlmin:  '<%= htmlmin.dist %>',
          url:    function(url) { return url.replace('.html', ''); },
          module: 'mpoApp'
          usemin: 'scripts\\scripts.js'
        }
      }
    },
<pre>
Running "uglify:generated" (uglify) task
[RangeError: Maximum call stack size exceeded]
>> Uglifying source .tmp/concat/scripts/scripts.js failed.
Warning: Uglification failed.
Maximum call stack size exceeded.
 Use --force to continue.

Aborted due to warnings.</pre>

**Comment by arnorhs** [comment]
i have the same issue. did you ever end up figuring it out @vijeetgv ?

**Comment by vijeetgv** [comment]
No, I skipped the on-the-fly usemin part & did it in 2 stages i.e. first grunt-react-templates & then usemin on the generated files. The *.rt.js files we can simply generate in the src folder itself (or maybe a subfolder, as desired) & then usemin them whenever we wish to build it to www.
